### DISCUSSION TITLE
Sprint 1: June 13-24 Check-ins

### DISCUSSION THREAD

**Discussion by eeholmes** [UW-Upwelling-Project/Summer-2022#10] [Team Posts]
Sprint 1: June 13-24. Project board: https://github.com/UW-Upwelling-Project/Summer-2022/projects/1

Post your daily checks on here: Keep it brief.
* What did you do yesterday
* What will you do today
* Are you stuck on anything.

In separate threads post anything you want to discuss. If we have ideas on anything anyone is stuck on, post a new thread on that.

**Comment by eeholmes** [comment]
June 13: Check-in

Eli:
* Yesterday: NA
* Today: Sending some emails, getting the first few background readings up
* Blocks: none

**Comment by XuweiyiChen** [comment]
Yesterday: NA
Today: Read 2 Bakun
Blocks: 
(1). The relationship between upwelling and global warming seems correlated, but in the Bakun from 2018 does not discuss the possibility of confounders.
(2). The second Bakun from 2015 is much harder to digest. Will read it again with more careful today.

**Comment by zhengx25-1976320** [comment]
Yesterday: self-learn ARIMA model. Figure out the topic of my project: I would like to use the time series model to study the intensity, location change, and seasonality of a upwelling system. 
Today: keep doing background reading, setting up jupyterhub.
Blocks:
(1) What does the E1, S1 mean in the project board.

**Comment by eeholmes** [comment]
E1 means Epic 1. Epic is a larger block of related work. S1 means Story associated with Epic 1. There would be multiple stories with epics so many S1's but only one E1.

**Comment by eeholmes** [comment]
**June 14** check-in

Eli

* Yesterday: Just organizing emails
* Today: Set up our video call for next week.
* Blocks: None

Note: I am on holiday this week so no more check-ins for me this week (and M-Tues have been minimal). Will be back on, next week.

**Comment by zhengx25-1976320** [comment]
Anthony
- Yesterday: read first Bakun and set up conda environment.
- Today: read second Bakun and additional readings. 
- Blocks: I don't have access to our bibliography. (denied by Zotero)

**Comment by XuweiyiChen** [comment]
Yesterday: Had a fever but read the second Bakun.
Today: read the second Bakun again and finish the SWOT tutorial.
Blocks:
(1). Errors due to one package: s3fs from SWOT tutorial.
(2). Will try to use AWS or look into the error.

**Comment by zhengx25-1976320** [comment]
Anthony

- Yesterday:Read the second Bakun.
- Today: finish up NASA Cloud tutorial, and read more papers.
- Blocks: I can't access Openscapes Jupyter Hub. https://openscapes.2i2c.cloud/hub/login?next=%2Fhub%2F

**Comment by XuweiyiChen** [comment]
Xuweiyi Chen

Yesterday: Read my notes about wavelets and set up the environment
Today: 
(1). Try some demos in R as well as NASA tutorial in python
(2). Spend a little more time with the second Bakun.
Blocks: 
(1). Should not upgrade R

**Comment by zhengx25-1976320** [comment]
Anthony
Yesterday: encounter problems when I follow tutorial. 

- Today: read _Under pressure: climate change, upwelling, and eastern boundary upwelling ecosystems._ and _The coastal winds off western subtropical South America in future climate scenarios_

Block:

- I can't access Openscapes Jupyter Hub. https://openscapes.2i2c.cloud/hub/login?next=%2Fhub%2F

**Comment by zhengx25-1976320** [comment]
> Xuweiyi Chen
> 
> Yesterday: Read my notes about wavelets and set up the environment
> Today:
> (1). Try some demos in R as well as NASA tutorial in python
> (2). Spend a little more time with the second Bakun.
> Blocks:
> (1). Should not upgrade R

Hi, William. Do you have access to Jupyter Hub?

**Comment by zhengx25-1976320** [comment]
Anthony
Yesterday: read .Under pressure: climate change, upwelling, and eastern boundary upwelling ecosystems. and The coastal winds off western subtropical South America in future climate scenarios

Today: read The coastal winds off western subtropical South America in future climate scenarios.
Block: None

**Comment by XuweiyiChen** [comment]
Xuweiyi Chen

Yesterday: NA
Today: (1). Google serveral questions from the second Bakun and read a little more about Hadley Cells. That part is particular interesting to me.
(2). Figure out some errors from last setup.
Block: Many questions but I feel like paper from Oceanology starts to get interesting.

**Comment by zhengx25-1976320** [comment]
Anthony
Yesterday: Found the combined effect of El Nino and multi-decadal climate variability on EBUS interesting. Maybe I can study a specific location especially affected by El Niño and La Niña events. 
Today: read tutorial about wavelet analysis and play with r code. 
Block: questions about how sea stratification counter upwelling

**Comment by zhengx25-1976320** [comment]
Anthony
Yesterday: read tutorial of wavelet analysis, and google wavelet transform, other related paper.
Today: keep learning wavelet analysis.
Block: I found it challenging and complex in general.

**Comment by XuweiyiChen** [comment]
Xuweiyi Chen

Yesterday: read Under Pressure: Climate Change, Upwelling, and Eastern Boundary Upwelling Ecosystems & Re-studied fourier theorm.
Today: start thinking about Hadley Cells and maybe related topics. Thinking about quantitify some results and apply causal inference for final judgments.
Block: Some terms are very hard to digest and understand. Some phenomenons are quite beyond my imagination specifically related to biological effects due to global warming but hadley cells related information are quite interesting.

**Comment by zhengx25-1976320** [comment]
Anthony
Yesterday: Study Fourier theory and wavelet analysis. I found https://ataspinar.com/2018/12/21/a-guide-for-using-the-wavelet-transform-in-machine-learning/ and https://ataspinar.com/2018/04/04/machine-learning-with-signal-processing-techniques/ pretty helpful.
Today: continue learning wavelet transform, meet up with Eli.
Block:None

**Comment by XuweiyiChen** [comment]
Xuweiyi Chen
Yesterday: Reviewed Fourier and some wavelet from  Data-Driven Science and Engineering. Read again one of the earlier paper about upwelling and low confidence issue of forecast.
Today: meet up with Eli & start thinking about questions that I can answer with data.
Block: None
